<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictEditWindow</class>
 <widget class="QDialog" name="DictEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>367</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>367</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>367</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>diskFind</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Редактирование словарей</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="curDictName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="wordsList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Выберите словари</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="CheckBoxList" name="cbxDictionary"/>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>320</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать выбранные словари</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBoxList</class>
   <extends>QComboBox</extends>
   <header location="global">checkboxlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
